cep-0011.md

```diff
@@ -233,15 +233,15 @@ See Addendum A below for a best effort in documenting it.
 
 ## References
 
-* [Rework linux/osx support plus new simplified format?](https://github.com/conda/menuinst/issues/48)
-* [Mamba's implementation in C++](https://github.com/mamba-org/mamba/blob/81a490a/src/core/menuinst.cpp)
-* [Interactions between conda, conda-standalone, constructor and menuinst](https://gist.github.com/jaimergp/7de5843421d63fa4a408ac5c8712c3c9)
-* [Change the API to `menuinst.install(path_or_dict)`](https://github.com/conda/menuinst/issues/25)
-* [`menuinst` wiki as of 2021.10.18](https://github.com/conda/menuinst/wiki/Menu-Shortcut-Config-Structure/632fbc84251c8a8093e1b56b0b5314d23c1e946b)
-* [freedesktop.org specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys)
-* [Core Foundation Keys (info.plist)](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/TP40009249-SW1)
-* [File type association in Windows](https://learn.microsoft.com/en-us/windows/win32/shell/fa-file-types)
-* [Default programs in Windows](https://learn.microsoft.com/en-us/windows/win32/shell/default-programs)
+- [Rework linux/osx support plus new simplified format?](https://github.com/conda/menuinst/issues/48)
```

Comment:
Why is using `*` for lists unacceptable? Both are valid list formats

Reply:
The linter requires that if you start with a given symbol, you have to use the same one all the way through the document.